### DISCUSSION TITLE
Support for Places API (New)?

### DISCUSSION THREAD

**Discussion by BenMaldoner** [SachinAgarwal1337/google-places-api#79] [Ideas]
Any plans for supporting [Places API (New) ](https://developers.google.com/maps/documentation/places/web-service/op-overview)?

**Comment by BenMaldoner** [comment]
It's already been discussed in the "issues" section I guess: https://github.com/SachinAgarwal1337/google-places-api/issues/76

**Comment by SachinAgarwal1337** [comment]
Support for Places API (New) has been released https://github.com/SachinAgarwal1337/google-places-api/releases/tag/3.0.0
